narf/combineutils.py

```diff
@@ -126,13 +126,13 @@ def __init__(self, filename, pseudodata=None):
             if "meta" in f.keys():
                 from narf.ioutils import pickle_load_h5py
                 self.metadata = pickle_load_h5py(f["meta"])
-                self.channel_axes = self.metadata["channel_axes"]
+                self.channel_info = self.metadata["channel_info"]
             else:
-                self.channel_axes = {
-                    "ch0": [hist.axis.Integer(0, self.nbins, underflow=False, overflow=False, name="obs")]
+                self.channel_info = {
+                    "ch0":{"axes": [hist.axis.Integer(0, self.nbins, underflow=False, overflow=False, name="obs")]}
                 }
                 if self.nbinsmasked > 0:
-                    self.channel_axes["ch1_masked"] = [hist.axis.Integer(0, self.nbinsmasked, underflow=False, overflow=False, name="masked")]
+                    self.channel_info["ch1_masked"] = {"axes": [hist.axis.Integer(0, self.nbinsmasked, underflow=False, overflow=False, name="masked")]}
```

Comment:
Would it make more sense for it to be called "ch0_masked"?

Reply:
Ah yes I think in the HDF5 writer we changed it to 0 but forget it here, I can can make another commit